sae/blocks.go

```diff
@@ -239,7 +249,7 @@ func (vm *VM) buildBlock(
 	}
 
 	var receipts types.Receipts
-	for _, b := range parent.WhenChildSettles(lastSettled) {
+	for _, b := range blocks.Range(parent.LastSettled(), lastSettled) {
```

Comment:
```suggestion
	settling := blocks.Range(parent.LastSettled(), lastSettled)
	for _, b := range settling {
```

See rationale above.